custom/inddex/food.py

```diff
@@ -0,0 +1,212 @@
+from .fixtures import FixtureAccessor
+
+MISSING = ''
+
+IN_UCR = 'in_ucr'
+IN_FOOD_FIXTURE = 'in_food_fixture'
+IS_RECALL_META = 'is_recall_meta'
+
+
+class I:
```

Comment:
I for Indicator?

Reply:
Yeah - I know single-letter class names are generally frowned upon, but I figured in this case things would be _more_ readable than repeating the full word 100 times.  Could also define the class as `Indicator` then immediately alias as `I`, or use plain tuples in the big list and generate the classes from them later.  I dunno, I think this way is fine.